internal/website/content/tutorials/guestbook.md

```diff
@@ -0,0 +1,308 @@
+---
+title: "Tutorial: Guestbook Application"
+linkTitle: "Guestbook"
+date: 2019-03-19T19:17:56-07:00
+weight: 2
+toc: true
+---
+
+In this tutorial, we will deploy an existing Go CDK application called Guestbook
+locally, then on Google Cloud Platform (GCP), Amazon Web Services (AWS), and/or
+Azure.
+
+<!--more-->
+
+Guestbook is a sample application that records visitors' messages and displays a
+cloud banner with an administrative message. The main business logic is written
+in a cloud-agnostic manner using MySQL, the generic blob API, and the generic
+runtimevar API. All platform-specific code is set up by
+[Wire](https://github.com/google/wire).
+
+## Prerequisites
+
+You will need to install the following software for this tutorial:
+
+-   [Git](https://git-scm.com/)
+-   [Go](https://golang.org/doc/install)
+-   [Wire](https://github.com/google/wire/blob/master/README.md#installing)
+-   [Docker Desktop](https://docs.docker.com/install/)
+-   [Terraform](https://www.terraform.io/intro/getting-started/install.html)
+
+[Terraform]: https://www.terraform.io/
```

Comment:
This looks like a left-over from some section that got moved.

Reply:
There were some links down below using it, but I don't think they were very useful since the user will already have installed it; removed them.